fs/xfstests.py.data/local.config

```diff
@@ -0,0 +1,4 @@
+export TEST_DEV=/dev/loop0
+export TEST_DIR=/mnt/test
+export SCRATCH_DEV=/dev/loop1
+export SCRATCH_MNT=/mnt/scratch
```

Comment:
@PraveenPenguin Okay, for now we can have this minimalistic version, with `TODO` added indicating creating FS, mounting the disk/loop devices, etc